theme/models.py

```diff
@@ -324,4 +325,36 @@ def force_unique_emails(sender, instance, **kwargs):
         if sender.objects.filter(username=username).exclude(pk=instance.id).exists():
             raise ValidationError("Username already in use.")
 
+
 pre_save.connect(force_unique_emails, sender=User)
+
+
+@receiver(models.signals.pre_save, sender=UserProfile)
```

Comment:
So this triggers anytime the UserProfile is updated?

Reply:
Yes, it is triggered anytime when user profile changes are saved. I cannot think of any more efficient way to achieve automatic deletion when a user profile CV or profile picture is updated without plugging in this trigger. I don't think user profiles are updated that often, so I don't think we need to worry about the performance, but let me know if you have better/more efficient ideas to achieve this automatic profile file cleanup.